index.js

```diff
@@ -33,11 +34,62 @@ Toolkit.run(async tools => {
     body: env.renderString(body, templateVariables),
     title: env.renderString(attributes.title, templateVariables)
   }
-
   tools.log.debug('Templates compiled', templated)
-  tools.log.info(`Creating new issue ${templated.title}`)
+
+  if (updateExisting) {
+    let existingIssue
+    tools.log.info(`Fetching issues issues with title "${templated.title}"`)
+    try {
+      existingIssue = await tools.github.issues.listForRepo({
+        ...tools.context.repo,
+        state: 'open'
+      }).find(existingIssue => {
+        return existingIssue.title === attributes.title && existingIssue.state === 'open'
+      })
```

Comment:
This API won't work if there are >20 issues, and the one to update isn't on the first page of the response. We can use [the Search API](https://developer.github.com/v3/search/#search-issues-and-pull-requests) instead:

```suggestion
      const searchResults = await tools.github.search.issuesAndPullRequests({
        q: `is:open is:issue repo:${process.env.GITHUB_REPOSITORY} title:${attributes.title}`
      })
```

Then some additional filtering on the results